### ISSUE TITLE
Add new functions

### ISSUE THREAD

**Issue by sandor111** [sdushantha/snapchat#3]
Hi maybe you can use this to add more features:
https://github.com/killed/Snapchat-Wiki/wiki

**Comment by sdushantha** [comment]
@sandor111 Thank you for letting me know about that repo!
I unforutnetly dont have much time these days to work on this module, but if anyone is wants to submit a PR, feel free :)

**Comment by thomas-milburn** [comment]
This repo is very helpful, I'm messing around with it atm. Thanks @sandor111

**Comment by sandor111** [comment]
lmk if there is any progress :), the header generation is prob the hardest part.

**Comment by thomas-milburn** [comment]
Yeah it doesn't mention anything about headers so can't really be used
